recipe/meta.yaml

```diff
@@ -1,36 +1,47 @@
 {% set name = "nglview" %}
-{% set version = "2.7.7" %}
-{% set sha256 = "fd84e20f0c1e183f2c429509c4a64b226e8591253545358062f48f3ec7dafddb" %}
+{% set version = "2.7.8" %}
+{% set sha256 = "47eb30bac596911211c847460e2457c7d4829f80d7a203550c67c4a3b16707a9" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://github.com/nglviewer/nglview/archive/refs/tags/v{{ version }}.tar.gz
   sha256: {{ sha256 }}
 
 build:
-  number: 1
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  number: 0
   noarch: python
 
 requirements:
   host:
-    - python >=3
+    - python >=3.6
     - pip
     - numpy
     - ipywidgets >=7
+    - jupyterlab
+    - jupyterlab_widgets
+    - jupyter-packaging
+    - nodejs
     - notebook
   run:
-    - python >=3
+    - python >=3.6
     - numpy
     - ipywidgets >=7
     - notebook
+    - jupyterlab_widgets
 
```

Comment:
`jupyterlab` should be an explicit dep I guess?

Reply:
It would be good to be able to install nglview without having to install jupyterlab. At the moment, [`jupyterlab_widgets` doesn't depend on `jupyterlab`](https://github.com/conda-forge/jupyterlab_widgets-feedstock/blob/81a59877060444da790647b4771f7671787de346/recipe/meta.yaml#L23) but [`ipywidgets` depends on `jupyterlab_widgets`](https://github.com/conda-forge/ipywidgets-feedstock/blob/1f4447619d3ce3403a0c594dcbc7ce3c6cb95aa4/recipe/meta.yaml#L28). I would say that this is fine as it is.

Reply:
> It would be good to be able to install nglview without having to install jupyterlab.

yeah.

Reply:
ok make sense.